cypress/support/login.ts

```diff
@@ -38,7 +38,7 @@ Cypress.Commands.add(
       cy.byLegacyTestID('login').should('be.visible');
       cy.task(
         'log',
-        ` Logging in as ${username || Cypress.env(BRIDGE_PASSWORD)}`
+        ` Logging in as ${Cypress.env(KUBEADMIN_USERNAME || username)}`
```

Comment:
```suggestion
        ` Logging in as ${Cypress.env(KUBEADMIN_USERNAME) || username}`
```
Instead of doing the check in 2 different places what about just 1 single check i.e.
`const inputUsername = Cypress.env(KUBEADMIN_USERNAME) || username` ?

Reply:
ACK.